### DISCUSSION TITLE
Web Base App

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#9] [Ideas]
First Idea would be a web based app. 

This would make it easier to maintain and would not have to develop separate apps for different operating systems (IOS/ANDROID) since they would all access via the web.

**Comment by judyn808** [comment]
This could be either a website or a Google form?

**Reply by JJ-Foust** [reply]
@judyn808 I brought up the idea to @ZhienWang and @tyliec about using Google Form and Google Sheet for the backend so that they would not have to program and SQL and save some time. They were looking into the feasbility of pulling that data from google to meet our needs

**Reply by judyn808** [reply]
Yes, saw that thread so was transferring the convo over here. If the system is really standalone, then the data collected should be kept very simple. Microchip, owner name, owner phone number? How do you think it can work if an owner had multiple pets?

**Reply by JJ-Foust** [reply]
No problem. Good back up moving it over.

**Comment by JJ-Foust** [comment]
Multiple Pet issue. 

I would either have the either add themselves to waitlist for each pet since they have separate MICROCHIPs
This could be because they might not be processed in the same batch since the animals have their own waitlist. 

Or just have them say they have multiple pets when they come in.
